configmaptocrs/README.md

```diff
@@ -92,3 +92,43 @@ status: {}
 ---
 
 ```
+
+## Run with ConfigMap Volume
+
+An approach from [this](https://github.com/metallb/metallb/issues/1527) issue, suggests running the configmaptocrs with:
+```
+kubectl -n metallb-system run configmaptocrs --restart=Never -it --rm --image overriden --overrides '
+{
+  "spec": {
+    "containers": [
+      {
+        "name": "configmaptocrs",
+        "image": "quay.io/metallb/configmaptocrs",
+        "command": [
+          "/configmaptocrs",
+          "-source",
+          "config",
+          "--data-only",
+        ],
+        "volumeMounts": [
+          {
+            "name": "config",
+            "mountPath": "/var/input"
+          }
+        ]
+      }
+    ],
+    "volumes": [
+      {
+        "name": "config",
+        "configMap": {
+          "name": "config"
+        }
+      }
+    ]
+  }
+}'
+```
+This will mount the MetalLB's config into the container's /var/input path, and print the result into stdout.
```

Comment:
I'd either document the full behaviour (from a user perspective) and / or the single flags meaning.
The full behaviour is "this will print the CRs corresponding to the configmap deployed on the cluster to the standard output".

Reply:
done